bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router';
 import { defineMessages, injectIntl } from 'react-intl';
+import { makeCall } from '/imports/ui/services/api';
+import Button from '/imports/ui/components/button/component';
 import Modal from '/imports/ui/components/modal/fullscreen/component';
+import Auth from '/imports/ui/services/auth';
+import userListService from '../user-list/service';
```

Comment:
As far as I understand @JaeeunCho just needs to move the function with makeCall to the service.js and then just import it from there directly to the component.